internal/uvm/create_lcow.go

```diff
@@ -291,11 +295,35 @@ func CreateLCOW(ctx context.Context, opts *OptionsLCOW) (_ *UtilityVM, err error
 				ImageFormat: imageFormat,
 			},
 		}
-		// Add to our internal structure
-		uvm.vpmemDevices[0] = &vpmemInfo{
-			hostPath: opts.RootFSFile,
-			uvmPath:  "/",
-			refCount: 1,
+		if uvm.vpmemMultiMapping {
+			pmem := newPackedVPMemDevice()
+			pmem.maxMappedDeviceCount = 1
```

Comment:
I'm kind of confused on this?

Reply:
OH I see, it's so we don't allocate anything else on the rootfs vpmem device. My bad :)